src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerEndpoint.cs

```diff
@@ -16,25 +16,19 @@ namespace Microsoft.Azure.Commands.TrafficManager.Models
 {
     using System;
     using Microsoft.Azure.Commands.TrafficManager.Utilities;
+    using Microsoft.Azure.Management.TrafficManager.Models;
 
-    public class Endpoint
+    public class TrafficManagerEndpoint
```

Comment:
That is a fair question. However I think the type is transparent for the users. They used to handle endpoints with the Endpoint property of TrafficManagerProfile type. Plus, I have only added new properties. Therefore it should not be. Correct me if I am wrong.